pages/index.js

```diff
@@ -51,6 +54,19 @@ export default function IndexPage(props) {
       setWorkflow(updatedWorkflow);
     })
   }
+  function updateWorkflowTaskFromMilestoneId(milestoneId) {
+    setloading(true);
+    makeApiRequest(
+      getMilestoneTask(
+        props.currentDatasetId,
+        milestoneId
+      )
+    ).then(({ data }) => {
+      const firstTaskIdInMilestone = data.tasks[0].id;
```

Comment:
@tomeksabala please confirm there will always be items in this list so that `data.tasks[0].id` would always work

Reply:
I imagine there might be a case to get a milestone with no tasks in it. You should check if tasks.length is non zero first.